### PR TITLE
Vcon 60 loading

### PR DESCRIPTION
**Pull request by shewood**
First part of the loading task. (no loading logic yet in this PR need to discuss further with those in the know) 
Updated some of the fonts to better match the Figma. 
Passed the icon for the bot and mail as a prop text for the inbox. 
( I updated the size of the message box so it matched the figma, I wasn't sure if that was needed however)

### CODE REVIEW

File: src/components/atoms/ReadNotification.vue [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+        pt-4
+      "
+    >
       {{ dayRead }}
     </p>
   </div>
-  <div v-else class="flex items-center justify-center h-full">
+  <div
+    v-else
+    class="
```
Comment (DaceyTom2) [inline src/components/atoms/ReadNotification.vue:21]: The little blue dot isn't in the centre anymore

File: src/components/molecules/Inbox.vue [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+    />
+    <!-- <inbox-item
+      tabindex="0"
+      dayRead="Thurs"
+      :icon="'src/assets/UserAccount/VirtualConcierge.svg'"
+    />
+    <inbox-item
+      tabindex="0"
+      :icon="'src/assets/UserAccount/VirtualConcierge.svg'"
+    /> -->
```
Comment (DaceyTom2) [inline src/components/molecules/Inbox.vue:26]: These can be removed

File: src/components/molecules/InboxItem.vue [hunk 3]
```diff
@@ -1,62 +1,59 @@
...
-          'text-sm md:text-base truncate overflow-ellipsis text-gray-dark',
+          'text-sm md:text-lg truncate overflow-ellipsis text-gray-dark pl-1',
         ]"
       >
-        Hi Mary, I'm your Virtual Con...
+        {{ bodyText }}
       </p>
     </div>
-    <div class="p-4 w-20 h-full">
+    <div>
```
Comment (DaceyTom2) [inline src/components/molecules/InboxItem.vue:35]: This div can be removed

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]
Looks good! a few small tweaks

File: src/components/atoms/ReadNotification.vue (hunk 1 above)
Comment (DaceyTom2) [reply src/components/atoms/ReadNotification.vue:21]: I missed a design meeting and am wrong

File: src/components/molecules/Inbox.vue (hunk 2 above)
Comment (shewood) [reply src/components/molecules/Inbox.vue:26]: Ok, I wasn't sure

**Review by DaceyTom2 — APPROVED** [review]

**Pull request merged**